blobfuse/blobfuse.cpp

```diff
@@ -468,11 +469,11 @@ void *azs_init(struct fuse_conn_info * conn)
         std::shared_ptr<OAuthTokenCredentialManager> tokenManager;
         if (config_options.caCertFile.empty())
         {
-            tokenManager = GetTokenManagerInstance(EmptyCallback);
+            tokenManager = GetTokenManagerInstance(EmptyCallback, config_options.debug_libcurl);
```

Comment:
Do we want to make it OAuth specific or rather a more generic option.

Reply:
it's not oauth specific. for all other rest calls we use the other http client anyway. this oauth option is for the initial token retrieval